File: LATIHAN1DPBO2023-main/PHP/Mahasiswa.php
<?php

// Class Mahasiswa yang berisi data diri umum mahasiswa, seperti
// -> Name, nim, jurusan, fakultas
class Mahasiswa {
	// Deklar Var private attr
	private $nim = '', 
					$name = '', 
					$jurusan = '', 
					$fakultas = '', 
					$foto_profile = '';

	// CONSTRUCT
	// Constructor kosong yang mengisi default value
	public function __construct($nim = '', $name = '', $jurusan = '', $fakultas = '', $foto_profile = '') {
		$this->nim = $nim;
		$this->name = $name;
		$this->jurusan = $jurusan;
		$this->fakultas = $fakultas;
		$this->foto_profile = $foto_profile;
	}

	// Setter dan Getter
	public function setNim($nim) { $this->nim = $nim; } // Method untuk menset atau mengubah atau menambah data NIM
	public function getNim() { return $this->nim; } // Method untuk mengambil data NIM

	public function setName($name) { $this->name = $name; } // Method untuk menset atau mengubah atau menambah data nama
	public function getName() { return $this->name; } // Method untuk mengambil data nama

  public function setJurusan($jurusan) { $this->jurusan = $jurusan; } // Method untuk menset atau mengubah atau menambah data jurusan
	public function getJurusan() { return $this->jurusan; } // Method untuk mengambil data jurusan

  public function setFakultas($fakultas) { $this->fakultas = $fakultas; } // Method untuk menset atau mengubah atau menambah data fakultas
	public function getFakultas() { return $this->fakultas; } // Method untuk mengambil data fakultas

	public function setFotoProfile($foto_profile) { $this->foto_profile = $foto_profile; } // Method untuk menset atau mengubah atau menambah data pathimg
	public function getFotoProfile() { return $this->foto_profile; } // Method untuk mengambil data pathimg

	// DESTRUCTOR
	public function __destruct() {}
}

?>

File: LATIHAN1DPBO2023-main/Python/Mahasiswa.py
# Class Mahasiswa yang berisi data diri umum mahasiswa, seperti
# -> Name, nim, jurusan, fakultas
class Mahasiswa :
  # Deklar private attr
  __name = ""
  __nim = ""
  __jurusan = ""
  __fakultas = ""
  
  # CONSTRUCTOR
  # Constructor kosong yang mengisi valuenya dengan default value atau string kosong
  def __init__ (self, nim = "", name = "", jurusan = "", fakultas = "") :
    self.__nim = nim
    self.__name = name
    self.__jurusan = jurusan
    self.__fakultas = fakultas
  
  # SETTER DAN GETTER
  # Method untuk mengambil data nama
  def get_name(self) :
    return self.__name
  # Method untuk menset atau mengubah atau menambah data nama
  def set_name(self, name) :
    self.__name = name;
  
  # Method untuk mengambil data nim
  def get_nim(self) :
    return self.__nim
  # Method untuk menset atau mengubah atau menambah data nim
  def set_nim(self, nim) :
    self.__nim = nim
  
  # Method untuk mengambil data jurusan
  def get_jurusan(self) :
    return self.__jurusan
  # Method untuk menset atau mengubah atau menambah data jurusan
  def set_jurusan(self, jurusan) :
    self.__jurusan = jurusan
  
  # Method untuk mengambil data fakultas
  def get_fakultas(self) :
    return self.__fakultas
  # Method untuk menset atau mengubah atau menambah data fakultas
  def set_fakultas(self, fakultas) :
    self.__fakultas = fakultas